examples/grpc.graphql

```diff
@@ -1,20 +1,21 @@
 # for test upstream server see [repo](https://github.com/tailcallhq/node-grpc)
 schema
   @server(port: 8000, graphiql: true)
-  @upstream(baseURL: "http://localhost:50051", httpCache: true, batch: {delay: 10}) {
+  @upstream(baseURL: "http://localhost:50051", httpCache: true, batch: {delay: 10})
+  @link(id: "news", src: "src/grpc/tests/news.proto", type: Protobuf) {
   query: Query
 }
 
 type Query {
-  news: NewsData! @grpc(service: "news.NewsService", method: "GetAllNews", protoPath: "src/grpc/tests/news.proto")
+  news: NewsData! @grpc(service: "news.NewsService", method: "GetAllNews", protoId: "news")
   newsById(news: NewsInput!): News!
-    @grpc(service: "news.NewsService", method: "GetNews", body: "{{args.news}}", protoPath: "src/grpc/tests/news.proto")
+    @grpc(service: "news.NewsService", method: "GetNews", body: "{{args.news}}", protoId: "news")
   newsByIdBatch(news: NewsInput!): News!
     @grpc(
       service: "news.NewsService"
       method: "GetMultipleNews"
       body: "{{args.news}}"
-      protoPath: "src/grpc/tests/news.proto"
+      protoId: "news"
```

Comment:
I feel we dont need protoId at all anymore. Since the buf files are loaded via link already we can just keep it simple by having a service, method and body. Infact we could also do just method. For eg:

**before**
```graphql
@grpc(service: "news.NewsService", method: "GetMultipleNews", body: "{{args.news}}", protoId: "news")
```

**After**
```graphql
@grpc(method: "news.NewsService.GetMultipleNews", body: "{{args.news}}")
```

IMO, this is a much nicer and a slicker user experience, makes me want to use the feature :)

Reply:
Feel free to create a separate issue if you think it's going out of scope :)

Reply:
Sure! #1101 created